pkg/porter/parameters.go

```diff
@@ -81,17 +81,14 @@ func (p *Porter) PrintParameters(ctx context.Context, opts ListOptions) error {
 		tp := dtprinter.DateTimePrinter{
 			Now: func() time.Time { return now },
 		}
-
-		printParamRow :=
-			func(v interface{}) []string {
-				cr, ok := v.(DisplayParameterSet)
-				if !ok {
-					return nil
-				}
-				return []string{cr.Namespace, cr.Name, tp.Format(cr.Status.Modified)}
-			}
-		return printer.PrintTable(p.Out, params, printParamRow,
-			"NAMESPACE", "NAME", "MODIFIED")
+		paramsSets := [][]string{}
+		for _, ps := range params[0].Parameters {
+			list := []string{}
+			list = append(list, ps.Name, ps.Source.Strategy, ps.Source.Hint, tp.Format(params[0].Status.Modified))
```

Comment:
Just a question about order of `ps.Source.Strategy` here and how it aligns columns `NAME", "VALUE", "STRATEGY", "MODIFIED`. It seems in the table columns it should come third after value but here it comes second after the name?

Reply:
Good catch, I can arrange the columns to match the requisite “value”. I can do this while changing strategy to TYPE as well as suggested in the other comment. 
Thanks @AGMETEOR